gradle.properties

```diff
@@ -1,13 +1,17 @@
 meritVersion=2.11-SNAPSHOT
-#gorm.version=7.0.8.RELEASE
-grailsVersion=4.1.2
-gormVersion=7.0.2
+grailsVersion=5.3.0
+groovyVersion=3.0.10
+gormVersion=7.3.2
+
 grailsWrapperVersion=1.0.0
 gradleWrapperVersion=5.0
-assetPipelineVersion=3.2.4
-mongoDBVersion=7.0
+assetPipelineVersion=3.4.4
+mongoDBVersion=7.3.2
```

Comment:
There isn't a grails mongo 7.3.2 - but this isn't used in MERIT so best to delete it.